examples/bri-3/src/bri/communication/agents/bpiMessages.agent.ts

```diff
@@ -39,7 +40,7 @@ export class BpiMessageAgent {
     );
 
     if (!bpiSubject) {
-      throw new NotFoundException(NOT_FOUND_ERR_MESSAGE);
+      throw new NotFoundException(BPI_SUBJECT_NOT_FOUND_ERR_MESSAGE);
```

Comment:
i dont know how was this working with wrong error message? will revisit tomorrow

Reply:
Probably message content is the same, that why the tests didn't fail